src/settings/settings_manager.cpp

```diff
@@ -102,7 +102,7 @@ const std::string SettingsManager::GetInfo() const {
   info.append(StringUtil::Format("%28s:   %-28s\n", "Index Tuner", GetBool(SettingId::index_tuner) ? "enabled" : "disabled"));
   info.append(StringUtil::Format("%28s:   %-28s\n", "Layout Tuner", GetBool(SettingId::layout_tuner) ? "enabled" : "disabled"));
   info.append(StringUtil::Format("%28s:   %-28s\n", "Code-generation", GetBool(SettingId::codegen) ? "enabled" : "disabled"));
-
+  info.append(StringUtil::Format("%28s:   %-28s\n", "Number of GC threads", GetInt(SettingId::gc_num_threads)));
```

Comment:
I don't think we want to have it be set from the command-line. Let's remove this.

Reply:
This is a normal GetInfo debug function, as far as I see; it has nothing to do with the option being available in command line.

Nevertheless, I don't think there's any way one can inhibit flags from appearing in ./peloton --help output and also setting them up from the command line. GFlags doesn't support that, I think (?)